### ISSUE TITLE
Entity Schema NOSAD

### ISSUE THREAD

**Issue by salgo60** [salgo60/NOSAD-POC-Wikidata#18]
* [Wikidata:WikiProject_Schemas](https://www.wikidata.org/wiki/Wikidata:WikiProject_Schemas)

**Fundering:** Idag är det lite otydligt tycker jag med strukturen på NOSAD datat - det blir bättre i Wikibase xxx
* Event
  * har namn
  * har datum tid 
  *.en eller flera inspelningar
  * en eller flera presentationer
  * en eller flera teman
* Presentation
  * görs av en eller flera talare
  * har en längd
  * har 0 eller flera presentationer/slides
  * handlar om ett eller flera ämnen
  * har en inspelning eller del av starttid
     * om del av inspelning har den
        * starttid
        * ev. sluttid
* Talare
   * har namn
   * har koppling organisation
   * har ev. koppling WD
   * har 0 eller flera kopplingar till arbetsfält
* Organisation 
   * har namn
   * har ev. koppling WD
   * har 0 eller flera kopplingar till arbetsfält

**Comment by jonassodergren** [comment]
Bra kämpat. Att skapa en "NOSAD persistent identifierare" för Event/Presentation/Slides är inte så krångligt vilket vi utvärderat i en annan tråd. Dagens lösning (ostrukturerad markdown) har sina brister =). Men den stora fördelen att den är enkel att underhålla vilket är extremt viktigt. Nästa steg med att publicera Event/Slides/Slides vill jag göra på ett sätt som inte kräver nya databaser, egenutvecklad mjukvara etc. Ska överväga om det kanske gå att utöka https://data.arbetsformedlingen.se/videos till https://data.arbetsformedlingen.se/events istället som innehåller inspelningar, event och presentationsbilder. Då kokar frågan ner till att ha en bra mappstruktur.

**Comment by salgo60** [comment]
> Att skapa en "NOSAD persistent identifierare" för Event/Presentation/Slides

@jonassodergren tror du skall ha egna NOSAD identifierare även för de ämnen som ett event handlar om sedan kan det ämnet referera andra externa vokabulär se #18 [#51](https://github.com/salgo60/DiggUptime/issues/51)

cc: tankar på det @gjonor [Oswald produktägaren](https://oswald.wikibase.cloud/wiki/Special:Statistics)

**Comment by dpriskorn** [comment]
> > Att skapa en "NOSAD persistent identifierare" för Event/Presentation/Slides
> 
> 
> 
> @jonassodergren tror du skall ha egna NOSAD identifierare även för de ämnen som ett event handlar om sedan kan det ämnet referera andra externa vokabulär se #18 [#51](https://github.com/salgo60/DiggUptime/issues/51) hur dessa kopplingar är kan då förtydligas med [SKOS Simple Knowledge Organization System Reference](https://www.w3.org/TR/skos-reference/) se exempel jag gjort med Skansens data [#11](https://github.com/salgo60/SkansenWikidata/issues/11)
> 
>   **"typ av mappning = [Property:P4390](https://www.wikidata.org/wiki/Property:P4390)"**
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/14206509/198528972-96e775f5-b86d-462e-a7f8-24faa555fed6.png)
> 
> 
> 
> * just detta att med externa kopplingar och koppla koncept kräver en helt annan nivå av samverkan kommunikation som Wikidata knappt har börjat fundera över....
> 
>    * vanligt är att 2 Wikipedia artiklar kopplas ihop på olika språk men sedan inser en som kan båda språken att den ena artikeln handlar om en vidare tolkning av ett begrepp en det andra språket....
> 
>       * [min tro som jag sa 2018](https://youtu.be/K0l4fv5uUvg?t=1577) att för att saker som detta skall ske behövs plattformar och [communication communication communication](https://agilecommsmanifesto.org/) och en helt annan laguppställning som vill kommunicera och eller har tydliga business case och då ser jag hur bra det fungera med att flytta pengar mellan banker där det var 0 tolerans för fel med semantisk interoperabilitet dvs. att Bank A inte fattar vilket konto Bank B vill sätta in pengar på.... samma vilja till kvalitet måste till med koncept/ord/ämnesord...
> 
>       
> 
> cc: tankar på det @gjonor [Oswald produktägaren](https://oswald.wikibase.cloud/wiki/Special:Statistics)
> 
> 
> 
> cc: @dpriskorn hade ni SKOS [Property:P4390](https://www.wikidata.org/wiki/Property:P4390)  på Internet Archive

Nej, men det är värt att lägga till. Ska fundera över vart det passar. 
Eftersom alla artikelobjekten är same as deras Wikidata qid så behövs det inte där. 
referenser är det bra att ha när vi matchar dem sedan mot Wikidata.

**Comment by salgo60** [comment]
Good lecture by Markus Krötzsch [start](https://youtu.be/ryxusv0s604?t=1105) - he was working with Denny on WIkidata 2012

* [at 20:10](https://youtu.be/ryxusv0s604?t=1218) he speaks about "vocabulary has meaning" --> I feel NOSAD vocabulary should be objects in oswald and use [SKOS](https://github.com/salgo60/SkansenWikidata/issues/11) with external vocabularies...

<img width="974" alt="image" src="https://user-images.githubusercontent.com/14206509/198939667-738cea27-738e-474a-bc01-b8b1513ff9d3.png">

**Comment by salgo60** [comment]
Test Oswald [E1](https://oswald.wikibase.cloud/wiki/EntitySchema:E1) som även kopplar #19 

Se även  **Data.europa.eu data quality guidelines** [doi:10.2830/333095](http://doi.org/10.2830/333095)  rekommendationer att bl.a använda SHACL [#81](https://github.com/salgo60/DiggUptime/issues/81)

![image](https://user-images.githubusercontent.com/14206509/199119783-25c42300-91ab-4e98-926f-b2d357d14eb6.png)
